dwitter/templates/base.html

```diff
@@ -37,23 +37,26 @@
       // Check for compatibility
       if (window.localStorage){
         var switchTheme = function(){
-          // Change the imported stylesheet from main.css to main_dark.css or vice versa
-          // Change localstorage theme to 'dark'
+          // Shift themes
           var theme = localStorage.getItem('theme');
-          if (theme == 'dark') {
-            localStorage.setItem('theme','');
-          } else {
+          if (theme === '') { 
```

Comment:
Make this:
```js
if (theme) {
```
then it will work as expected when theme is `''` and also when theme is `undefined` (which is what happens the very first time.)

Reply:
✅